tuning/data/setup_dataprocessor.py

```diff
@@ -231,6 +233,42 @@ def _get_default_dataset_handlers(data_args, tokenizer_kwargs):
     return [handler], data_args.dataset_text_field
 
 
+### Vsion Data Format
+def _get_vision_dataset_handlers(data_args, processor_kwargs):
+
+    handlers = []
+
+    # First data handler configuration
+    handler_fn_kwargs1 = {
+        "dataset_text_field": data_args.dataset_text_field,
+        "conversation_column": data_args.dataset_text_field,
+    }
+    handler_kwargs1 = {
+        "fn_kwargs": handler_fn_kwargs1,
+        "remove_columns": None,
+    }
+    handlers.append(
+        DataHandlerConfig("apply_tokenizer_chat_template", arguments=handler_kwargs1)
+    )
+
+    # Second data handler configuration
+    fn_kwargs2 = {
```

Comment:
```suggestion
    handler_fn_kwargs2 = {
```

and `handler_kwargs2` like above.